src/lib/message/L1ToL2Message.ts

```diff
@@ -307,34 +308,37 @@ export class L1ToL2MessageReader extends L1ToL2Message {
     // to do this we need to filter through the whole lifetime of the ticket looking
     // for relevant redeem scheduled events
     if (creationReceipt) {
-      const iFace = ArbRetryableTx__factory.createInterface()
-      const redeemTopic = iFace.getEventTopic('RedeemScheduled')
-
       let increment = 1000
       let fromBlock = await this.l2Provider.getBlock(
         creationReceipt.blockNumber
       )
       const creationBlock = await this.l2Provider.getBlock(
```

Comment:
I think you can get rid of the creationBlock var now

Reply:
nice catch, pushed the fix